### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by diidiiman**
Vault view, recipient view, login view.

### CODE REVIEW

File: src/app/dashboard/vaults/show/show.component.ts [hunk 1]
```diff
@@ -32,53 +21,50 @@ import { VaultI } from '@t/Vault';
...
   private isTrezorLogged;
-  private billingInterval;
   private datatableClasses = {
     pagerLeftArrow: 'fa fa-angle-left',
     pagerRightArrow: 'fa fa-angle-right',
     pagerPrevious: 'fa fa-angle-double-left',
     pagerNext: 'fa fa-angle-double-right'
   };
 
+  private fragment: string;
```
Comment (EduardsE) [inline src/app/dashboard/vaults/show/show.component.ts]: This variable name will not mean anything to the reader.

File: src/app/dashboard/vaults/show/show.component.ts [hunk 2]
```diff
@@ -87,12 +73,110 @@ export class VaultsShowComponent implements OnInit, OnDestroy {
...
+      this.vault['passwordEntry'] = '';
+
+      this.unlockVaultWithDataAndKey(
+        res['vaultLockCiphertext'], this.vault['password']
+      );
+
+      await this.getFilesAndData();
+
+    } catch (httpError) {
+      console.log(httpError);
```
Comment (EduardsE) [inline src/app/dashboard/vaults/show/show.component.ts]: No need for this. errorHandler.serverErro should already console.log the error.

File: src/assets/stylesheets/main.scss [hunk 3]
```diff
@@ -80,7 +89,7 @@ body {
 }
 
 .block-ui-wrapper {
-  background: rgba(1, 95, 182, 0.5) !important;
+  background: rgba(33, 21, 81, 0.5) !important;
```
Comment (EduardsE) [inline src/assets/stylesheets/main.scss]: Color variables.

File: src/app/dashboard/vaults/show/show.component.ts (hunk 1 above)
Comment (diidiiman) [reply src/app/dashboard/vaults/show/show.component.ts]: Example of necessity for the comments in the code. "Fragments" is self-explanatory :P
(https://angular.io/guide/router#query-parameters-and-fragments)
So, if this seems wierd, then maybe we should all stick to commenting our code.

**Comment by diidiiman** [comment]
Valid comments. Still shedding that rust off.

File: src/app/dashboard/top-bar/top-bar.component.jade [hunk 4]
```diff
@@ -7,32 +7,30 @@
...
+            i.fas.fa-cogs
+            | Settings
+
+          a.dropdown-item(routerLink="/dashboard/billing/status")
+            i.fas.fa-credit-card
+            | Billing
+
+          .dropdown-divider
+
+          a.dropdown-item((click)="logOut()") Exit
```
Comment (EduardsE) [inline src/app/dashboard/top-bar/top-bar.component.jade]: No icon for logout? :(

File: src/app/dashboard/vaults/intro/intro.component.jade [hunk 5]
```diff
@@ -1,17 +0,0 @@
...
-  .row
-    .no-vaults.col-sm-12
-      h2.cta-title
-        | It seems that you do not have a vault set up.
-      .btn.btn-lg.btn-success((click)="showNewVaultForm()", [hidden]="additionalContent")
-        | Click here to get started
-
-      .more-details.mt-3(*ngIf="additionalContent")
-        h3
-          | Please note that the passphrase is used to encrypt all the communication you perform with DigiPulse service and the passphrase is never exchanged with our servers. Also - it is not possible to change the encryption key later on, so choose something that you can remember.
```
Comment (EduardsE) [inline src/app/dashboard/vaults/intro/intro.component.jade]: Can we stick to shorter lines?

File: src/app/dashboard/vaults/show/file-upload/file-upload.component.scss [hunk 6]
```diff
@@ -120,7 +121,7 @@ div#dropzone {
   display: table;
   color: #fff;
   background: rgba(0, 0, 0, 0.45);
-  font: bold 42px Lato;
+  font: bold 42px "Roboto";
```
Comment (EduardsE) [inline src/app/dashboard/vaults/show/file-upload/file-upload.component.scss]: :( :( :(

**Pull request merged**